contents.json

```diff
@@ -4866,6 +4866,13 @@
       "description": "A command line interface for your Swift scripts.",
       "homepage": "https://github.com/yonaskolb/Beak"
     }, {
+      "title": "CodableWrappers",
+      "category": "misc",
+      "description": "A Collection of PropertyWrappers to make custom Serialization of Swift Codable Types easy.",
```

Comment:
@GottaGetSwifty Please remove 'Swift' word from title.

Reply:
The title is CodableWrappers?

Reply:
Oh, I'm sorry. I mean description.

Reply:
👍 done

Reply:
Thank you! 👍